Page/CertificatePage.cs

```diff
@@ -0,0 +1,188 @@
+﻿
+using AventStack.ExtentReports;
+using NunitCompetition.Utilities;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System.Runtime.ConstrainedExecution;
+using System.Text.Json;
+
+namespace NunitCompetition.Page
+{
+    public class CertificatePage : CommonHook
+    {
+        public static void GoToTab(IWebDriver driver)
+        {
+            // Navigating to Certificate tab
+            IWebElement CertificateTab = driver.FindElement(By.XPath("//*/div/section[2]/div/div/div/div[3]/form/div[1]/a[4]"));
```

Comment:
you need to use the POM pattern your web elements should be reusable via instance variables